README.md

```diff
@@ -106,8 +108,29 @@ Set to `debug` enable debug los from the Datadog Lambda Layer.
 
 Generate enhanced Datadog Lambda integration metrics, such as, `aws.lambda.enhanced.invocations` and `aws.lambda.enhanced.errors`. Defaults to true.
 
+### DD_LAMBDA_HANDLER
+
+For use with the [redirected handler](#Redirected-Handler) method. Location of your original lambda handler.
+
+### DD_TRACE_ENABLED
+
+When used with the [redirected handler](#Redirected-Handler) method, will auto initialize the tracer when set to true.
+
 ## Basic Usage
 
+Datadog needs to be able to read headers from the incoming Lambda event. To do this, you must wrap your handler function with our library. We provide some easy ways of wrapping your handlers.
+
+### Redirected Handler
+
+We provide a swap in replacement handler, with zero required code changes.
+
+1. Set the environment variable `DD_LAMBDA_HANDLER` to your regular handler location, eg. `myfunc.handler`.
+2. If using the layer, set your handler to `datadog_lambda.handler.handler`.
```

Comment:
Don't you **always** need to set handler to `datadog_lambda.handler.handler`?

Reply:
Good point, I'll fix